addons/mail/static/tests/discuss_app/jump_to_present_tests.js

```diff
@@ -46,13 +39,40 @@ QUnit.test("Basic jump to present when scrolling to outdated messages", async (a
     await contains(".o-mail-Thread", { scroll: "bottom" });
 });
 
-QUnit.test("Jump to old reply should prompt jump to presence", async () => {
-    // make scroll behavior instantaneous.
-    patchWithCleanup(Element.prototype, {
-        scrollIntoView() {
-            return super.scrollIntoView(true);
-        },
-    });
+QUnit.test(
+    "Basic jump to present when scrolling to outdated messages (chatter, DESC)",
+    async (assert) => {
+        const pyEnv = await startServer();
+        const partnerId = pyEnv["res.partner"].create({ name: "Demo User" });
+        for (let i = 0; i < 20; i++) {
+            pyEnv["mail.message"].create({
+                body: "Non Empty Body ".repeat(100),
+                message_type: "comment",
+                model: "res.partner",
+                res_id: partnerId,
+            });
+        }
+        const { openFormView } = await start();
+        await openFormView("res.partner", partnerId);
+        await contains(".o-mail-Message", { count: 20 });
+        await contains(".o-mail-Thread");
+        assert.ok(
+            document.querySelector(".o-mail-Thread").scrollHeight > PRESENT_THRESHOLD,
+            "should have enough scroll height to trigger jump to present"
+        );
+        await contains(".o-mail-Thread", { scroll: 0 });
+        await scroll(".o_content", "bottom");
```

Comment:
Why does it work with `o_content` here? Shouldn't chatter have its own scrollbar?

Reply:
In non-XXL size, the chatter is at bottom and share scrollable with the page, therefore `o_content`

Reply:
In previous versions we specifically avoided making changes to scroll value when the chatter is at bottom because it makes little sense.

We don't want to scroll restore to 0, but to the top of the thread, which might be completely different.

Reply:
I'd prefer an XXL size test and ignore the small one.

Reply:
Done